src/webview/common/searchBar.tsx

```diff
@@ -15,30 +15,43 @@ const useSearchBarStyle = makeStyles(searchBarStyle);
 export const SearchBar: React.FC<SearchBarProps> = ({
     title,
     onSearchBarChange,
-    searchBarValue
+    searchBarValue,
+    resultCount
 }: SearchBarProps) => {
     const searchBarStyle = useSearchBarStyle();
+    const icon = searchBarValue.length > 0 ?
+        <InputAdornment position='end'>
+            <IconButton aria-label='close' onClick={(): void => onSearchBarChange('')}>
+                <CloseRounded />
+            </IconButton>
+        </InputAdornment> : undefined;
     return (
-        <div className={searchBarStyle.searchBar}>
-            <TextField
-                className={searchBarStyle.searchBarInput}
-                placeholder={title}
-                value={searchBarValue}
-                onChange={(e) => {
-                    onSearchBarChange(e.target.value);
-                }}
-                sx={{
-                    '& fieldset': { border: 'none' }
-                }}
-                onClick={(): void => onSearchBarChange('')}
-                InputProps={{
-                    startAdornment: (
-                        <InputAdornment position='start' style={{ paddingLeft: '0.0625rem' }}>
-                            <SearchRounded />
-                        </InputAdornment >
-                    )
-                }}
-            />
-        </div>
+        <Container maxWidth='md'>
+            <div className={searchBarStyle.searchBar}>
+                <TextField
+                    className={searchBarStyle.searchBarInput}
+                    placeholder={title}
+                    value={searchBarValue}
+                    onChange={(e) => {
+                        onSearchBarChange(e.target.value);
+                    }}
+                    sx={{
+                        '& fieldset': { border: 'none' }
+                    }}
+                    InputProps={{
+                        startAdornment: (
+                            <InputAdornment position='start'>
+                                <SearchRounded />
+                            </InputAdornment >
+                        ),
+                        endAdornment: (icon)
+                    }}
+                    fullWidth
+                />
+                <Typography variant='caption' style={{ padding: '0.5rem 0', fontSize: '0.7rem' }}>
+                    {resultCount > 0 ? `1-${resultCount} of ${resultCount}` : `0 of 0`}
```

Comment:
Add more right margin to the result text, so that it does not look very close to the search bar.

If no results are there, instead of saying `0 of 0`, we need to mention `No results found`.